package.json

```diff
@@ -102,22 +98,18 @@
 		"@testing-library/svelte": "^4.0.3",
 		"@testing-library/user-event": "^14.4.3",
 		"@types/jest-axe": "^3.5.5",
-		"@types/prettier": "^2.7.3",
-		"@types/testing-library__jest-dom": "^5.14.9",
```

Comment:
These are stub type definitions, meaning that the dependencies themselves are already providing type definitions nowadays.